docs/product-user-guide/src/main/asciidoc/data-sets-caching-con.adoc

```diff
@@ -9,14 +11,12 @@
 [float]
 == Client cache
 
-When this cache is turned on, the data set is cached in a web browser during the look-up operation.
-Consequently, further look-up operations do not perform requests to the back end.
+When this cache is turned on, the data set is cached in a web browser during the look-up operation. Consequently, further look-up operations do not perform requests to the back end. Data set operations like grouping, aggregations, filtering and sorting are processed in the web browser. Enable client caching only if the data set size is small otherwise for large data sets, browser issues such as slow performance or intermittent freezing can occur.
```

Comment:
... "if the data set size is small otherwise..." => if the data set size is small (for example, less than 10Mb is a good comprimise) otherwise

Reply:
what above this? ^

Reply:
Change acknowledged.